pkg/util/workloadrepo/worker.go

```diff
@@ -174,13 +175,22 @@ func init() {
 	})
 }
 
-func initializeWorker(w *worker, etcdCli *clientv3.Client, newOwner func(string, string) owner.Manager, sesspool sessionPool) {
+func initializeWorker(w *Worker, etcdCli *clientv3.Client, newOwner func(string, string) owner.Manager, sesspool sessionPool) {
 	w.etcdClient = etcdCli
 	w.sesspool = sesspool
 	w.newOwner = newOwner
 	w.wg = util.NewWaitGroupEnhancedWrapper("workloadrepo", nil, false)
 }
 
+// TakeSnapshot will trigger a snapshot immediately if possible.
+func TakeSnapshot() error {
+	if workerCtx.snapshotChan == nil {
+		return errors.New("workload repository is not enabled yet")
```

Comment:
```suggestion
		return errors.New("Workload repository is not enabled yet")
```
Since this message will be sent to the client, it's better to have the 1st character in upper case.